tests/rptest/services/redpanda.py

```diff
@@ -2513,6 +2514,13 @@ def __init__(self,
         self._schema_registry_config = schema_registry_config
         self._audit_log_config = audit_log_config
         self._failure_injection_enabled = False
+        # Tolerate redpanda nodes not running during health checks. This is
+        # useful when running i.e. with node_operations.FailureInjectorBackgroundThread
+        # which can kill redpanda nodes.
+        # This is a number to allow multiple callers to set it.
+        self.tolerate_not_running_requests = 0
```

Comment:
nit: maybe we can think about better name here ?, this name seems to be confusing, maybe something `tollerate_not_running_nodes` ?

Reply:
i like this ^